### DISCUSSION TITLE
Please return to the administrative section as it was the method of loading new models.

### DISCUSSION THREAD

**Discussion by SoftUniq** [open-webui/open-webui#7099] [General]
# Feature Request

**Is your feature request related to a problem? Please describe.**  
After the update, the ability to add and download new models, as well as the functionality for uploading guff models, has disappeared. This causes inconvenience, as these features previously made working with models much easier.

**Describe the solution you'd like**  
I would like to see the ability to add, download new models, and upload guff models restored to the administrative section. This would bring back the convenient way of working with models that was available before the update.

**Describe alternatives you've considered**  
Currently, alternative methods do not provide the same level of convenience. It requires manually uploading models through workarounds, which is less efficient and user-friendly.

**Additional context**  
Please consider restoring these features to the administrative section to enable users to work more productively with the available functionality.

**Comment by tjbck** [comment]
![image](https://github.com/user-attachments/assets/35ce97da-3434-4267-a0d6-8da04485f5e1)

Please read the changelogs.

![image](https://github.com/user-attachments/assets/4ec87b04-742b-40ca-be03-2e1eb37aab5a)

![image](https://github.com/user-attachments/assets/f0912ae8-b944-40b0-861b-4acacf94c415)

**Reply by SoftUniq** [reply]
Uh...sorry but I really read and didn't find it =) Thanks, now everything is clear and in its place.
